classes/ai_manager_utils.php

```diff
@@ -526,7 +526,7 @@ public static function determine_purposes_availability(stdClass $user, int $cont
                 $purposes[] = [
                     'purpose' => $purpose,
                     'available' => self::AVAILABILITY_DISABLED,
-                    'errormessage' => get_string('error_purposenotconfigured', 'local_ai_manager'),
+                    'errormessage' => get_string('error_purposenotconfigured', 'local_ai_manager', $purpose),
```

Comment:
There are more that just this appearance of the lang string, please add the parameter everywhere it is being used